### PR TITLE
Added docs for exporting a runbook

### PR DESCRIPTION
**Pull request by mcasperson**
This PR adds documentation for the process of exporting and importing runbooks.

### CODE REVIEW

File: src/pages/docs/platform-engineering/managing-runbook-resources.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2023-01-01
+modDate: 2023-01-01
```
Comment (harrisonmeister) [inline src/pages/docs/platform-engineering/managing-runbook-resources.md:4]:
```suggestion
pubDate: 2023-11-09
modDate: 2023-11-09
```

File: src/pages/docs/platform-engineering/managing-runbook-resources.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+:::
+
+Runbooks can be defined in a Terraform module in two ways:
+
+* Write the module by hand
+* Serialize an existing project to a Terraform module with [octoterra](https://github.com/OctopusSolutionsEngineering/OctopusTerraformExport)
+
+## Writing by hand
+
+The process of defining a runbook in Terraform is much the same as defining a project. Both a runbook and a project have the concept of a deployment process that defines the steps to be run. See the [Managing project resources](managing-project-resources) section for more information on defining steps in Terraform by hand.
```
Comment (harrisonmeister) [inline src/pages/docs/platform-engineering/managing-runbook-resources.md:27]: I _think_ this is the correct syntax (not 100% sure though :smile:) 
```suggestion
The process of defining a runbook in Terraform is much the same as defining a project. Both a runbook and a project have the concept of a deployment process that defines the steps to be run. See the [Managing project resources](/docs/platform-engineering/managing-project-resources) section for more information on defining steps in Terraform by hand.
```

File: src/pages/docs/platform-engineering/managing-runbook-resources.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+The zip file has one directory called `space_population` which contains a Terraform module to populate a space with the exported resources.
+
+:::div{.hint}
+Many of the exported resources expose values, like resource names, as Terraform variables with default values. You can override these variables when applying the module to customize the resources, or leave the Terraform variables with their default value to recreate the resources with their original names.
+:::
+
+## Dealing with project variables
+
+The exported module defines only the runbook and the runbook deployment process. It does not define any other project level resources like project variables or library variable sets. Any project that the exported runbook is added to is expected define all the variables referenced by the runbook.
```
Comment (harrisonmeister) [inline src/pages/docs/platform-engineering/managing-runbook-resources.md:68]:
```suggestion
The exported module defines only the runbook and the runbook deployment process. It does not define any other project-level resources like project variables or library variable sets. Any project that the exported runbook is added to is expected to define all the variables referenced by the runbook.
```

File: src/pages/docs/platform-engineering/managing-runbook-resources.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+Any project level variables required by the runbook can be defined as Terraform resources and deployed alongside the exported runbook module. The instructions documented in the [Managing project resources](managing-project-resources) section can be used to export a project to a Terraform module. The project level variables can be copied out of the exported project module as needed and placed in their own module as needed.
```
Comment (harrisonmeister) [inline src/pages/docs/platform-engineering/managing-runbook-resources.md:70]:
```suggestion
Any project level variables required by the runbook can be defined as Terraform resources and deployed alongside the exported runbook module. The instructions documented in the [Managing project resources](/docs/platform-engineering/managing-project-resources) section can be used to export a project to a Terraform module. The project level variables can be copied out of the exported project module as needed and placed in their own module as needed. 
```

File: src/pages/docs/platform-engineering/managing-runbook-resources.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+Executing the runbook will create a new runbook in an existing project. Any space level resources referenced by the project are resolved by the resource name using Terraform [data sources](https://developer.hashicorp.com/terraform/language/data-sources), so the runbook can be imported into any space with the correctly named space level resources.
+
+### Updating project resources
+
+The runbooks `__ Serialize Runbook` and `__ Deploy Runbook` can be run as needed to serialize any changes to the upstream runbook and deploy the changes to downstream runbooks. The Terraform module zip file pushed to the built-in feed is versioned with a unique value each time, so you can also revert changes by redeploying an older package.  In this way you can use Octopus to deploy Octopus runbooks using the same processes you use Octopus to deploy applications.
```
Comment (harrisonmeister) [inline src/pages/docs/platform-engineering/managing-runbook-resources.md:96]:
```suggestion
The runbooks `__ Serialize Runbook` and `__ Deploy Runbook` can be run as needed to serialize any changes to the upstream runbook and deploy the changes to downstream runbooks. The Terraform module zip file pushed to the built-in feed is versioned with a unique value each time, so you can also revert changes by redeploying an older package.  In this way, you can use Octopus to deploy Octopus runbooks using the same processes you use Octopus to deploy applications. 
```

**Review by harrisonmeister — COMMENTED** [review]
Made a few minor suggestions, but otherwise LGTM!

File: src/pages/docs/platform-engineering/managing-runbook-resources.md (hunk 2 above)
Comment (mcasperson) [reply src/pages/docs/platform-engineering/managing-runbook-resources.md:27]: Either syntax appears to work. This is a screenshot of a published page, and it uses the relative path for a sibling page, which publishes the correct link when the markdown is compiled.

image

Comment (mcasperson) [reply src/pages/docs/platform-engineering/managing-runbook-resources.md:27]: I'll stick with the simpler syntax for now as that it how the other pages define links between themselves.

File: src/pages/docs/platform-engineering/managing-runbook-resources.md (hunk 4 above)
Comment (mcasperson) [reply src/pages/docs/platform-engineering/managing-runbook-resources.md]: :+1: I'll add those suggestions, but leave the link with the short relative url as that is how links in other pages have been defined.

File: src/pages/docs/platform-engineering/managing-runbook-resources.md (hunk 2 above)
Comment (harrisonmeister) [reply src/pages/docs/platform-engineering/managing-runbook-resources.md:27]: Thanks for checking/confirming - good to know!

**Review by harrisonmeister — APPROVED** [review]
👍

**Pull request merged**